plugins/commands/remove.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const baseSchema = schema.models.command.base;
+const req = require('request');
+
+/**
+ * Remove command from store and API
+ * @method removeCommand
+ * @param  {Object}   command         The Command object
+ * @param  {String}   storeUrl        URL to the store
+ * @param  {String}   authToken       Bearer Token to be passed to the store
+ * @return {Promise}
+ */
+function removeCommand(command, storeUrl, authToken) {
```

Comment:
You can put `function removeCommand({namespace, name, version}, storeUrl, authToken)` here

Reply:
There's a `command.remove()` later on in the function